CI build failing because it can't find browser-upgrade.html

```
Liquid Exception: Could not locate the included file 'browser-upgrade.html' in any of ["/srv/jekyll/_includes", "/tmp/jekyll-remote-theme-20220606-1-f1nf59/_includes"]. Ensure it exists in one of those directories and is not a symlink as those are not allowed in safe mode. in /_layouts/default.html
jekyll 3.9.2 | Error:  Could not locate the included file 'browser-upgrade.html' in any of ["/srv/jekyll/_includes", "/tmp/jekyll-remote-theme-20220606-1-f1nf59/_includes"]. Ensure it exists in one of those directories and is not a symlink as those are not allowed in safe mode.
/usr/gem/gems/jekyll-3.9.2/lib/jekyll/tags/include.rb:121:in `locate_include_file': Could not locate the included file 'browser-upgrade.html' in any of ["/srv/jekyll/_includes", "/tmp/jekyll-remote-theme-20220606-1-f1nf59/_includes"]. Ensure it exists in one of those directories and is not a symlink as those are not allowed in safe mode. (IOError)
	from /usr/gem/gems/jekyll-include-cache-0.2.1/lib/jekyll-include-cache/tag.rb:31:in `path'
	from /usr/gem/gems/jekyll-include-cache-0.2.1/lib/jekyll-include-cache/tag.rb:12:in `render'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/block_body.rb:103:in `render_node_to_output'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/block_body.rb:91:in `render'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/template.rb:208:in `block in render'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/template.rb:242:in `with_profiling'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/template.rb:207:in `render'
	from /usr/gem/gems/liquid-4.0.3/lib/liquid/template.rb:220:in `render!'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/liquid_renderer/file.rb:30:in `block (2 levels) in render!'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/liquid_renderer/file.rb:42:in `measure_bytes'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/liquid_renderer/file.rb:29:in `block in render!'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/liquid_renderer/file.rb:49:in `measure_time'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/liquid_renderer/file.rb:28:in `render!'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/renderer.rb:126:in `render_liquid'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/renderer.rb:[195](https://github.com/Cambridge-ICCS/cambridge-iccs.github.io/runs/6756145877?check_suite_focus=true#step:3:196):in `render_layout'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/renderer.rb:158:in `place_in_layouts'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/renderer.rb:88:in `render_document'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/renderer.rb:62:in `run'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:479:in `render_regenerated'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:464:in `block (2 levels) in render_docs'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:463:in `each'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:463:in `block in render_docs'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:462:in `each_value'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:462:in `render_docs'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:191:in `render'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/site.rb:71:in `process'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/command.rb:28:in `process_site'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/build.rb:65:in `build'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/build.rb:36:in `process'
	from /usr/gem/gems/jekyll-3.9.2/lib/jekyll/commands/build.rb:18:in `block (2 levels) in init_with_program'
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:[220](https://github.com/Cambridge-ICCS/cambridge-iccs.github.io/runs/6756145877?check_suite_focus=true#step:3:221):in `block in execute'
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `each'
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/command.rb:220:in `execute'
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary/program.rb:42:in `go'
	from /usr/gem/gems/mercenary-0.3.6/lib/mercenary.rb:19:in `program'
	from /usr/gem/gems/jekyll-3.9.2/exe/jekyll:15:in `<top (required)>'
	from /usr/local/bundle/bin/jekyll:27:in `load'
	from /usr/local/bundle/bin/jekyll:27:in `<main>'
Error: Process completed with exit code 1.
```

--- a/_layouts/default.html
+++ b/_layouts/default.html
@@ -16,7 +16,6 @@
 
   <body class="layout--{{ page.layout | default: layout.layout }}{% if page.classes or layout.classes %}{{ page.classes | default: layout.classes | join: ' ' | prepend: ' ' }}{% endif %}">
     {% include_cached skip-links.html %}
-    {% include_cached browser-upgrade.html %}
     {% include_cached masthead.html %}
     <div class="initial-content">
       {{ content }}

--- a/_layouts/index.html
+++ b/_layouts/index.html
@@ -16,7 +16,6 @@
 
   <body class="layout--{{ page.layout | default: layout.layout }}{% if page.classes or layout.classes %}{{ page.classes | default: layout.classes | join: ' ' | prepend: ' ' }}{% endif %}">
     {% include_cached skip-links.html %}
-    {% include_cached browser-upgrade.html %}
     {% include_cached masthead.html %}
 
     <div class="initial-content">

